app/components/views/Send.js

```diff
@@ -111,6 +111,10 @@ class Send extends Component{
         updatedOutputErrors[i].amountError = '*Please enter a valid amount (> 0)';
         checkErrors = true;
       }
+      if (updatedOutputErrors[i].amount > 9223372036854776000) {    // 2^63
```

Comment:
Why not use Number.MAX_SAFE_INTEGER instead here?

Usually I don't like to use magic numbers like that